private-templates-service/client/public/containers/adaptivecards-defaulthost.css

```diff
@@ -0,0 +1,205 @@
+.card-frame {
```

Comment:
You didn't find a way to webpack these?

Reply:
As a temporary solution I thought we could just copy the files into the public folder and address the webpack issue later on.